sql/core/src/main/scala/org/apache/spark/sql/jdbc/JdbcDialects.scala

```diff
@@ -240,8 +240,31 @@ abstract class JdbcDialect extends Serializable with Logging{
         getJDBCType(dataType).map(_.databaseTypeDefinition).getOrElse(dataType.typeName)
       s"CAST($l AS $databaseTypeDefinition)"
     }
+
+    override def visitSQLFunction(funcName: String, inputs: Array[String]): String = {
+      if (isSupportedFunction(funcName)) {
+        s"""$funcName(${inputs.mkString(", ")})"""
+      } else {
+        throw QueryCompilationErrors.noSuchFunctionError(dialectName, funcName)
```

Comment:
This is not a user-facing error, right? The framework will catch the error and give up the pushdown. I think just throwing `UnsupportedException` is good enough here, with a comment to explain how it will be handled and is not user-facing.